### ISSUE TITLE
Kreya becomes unresponsive after receiving large number of responses from streaming method

### ISSUE THREAD

**Issue by agrounds** [riok/Kreya#135]
**Describe the bug**
When calling a streaming method that returns thousands of small response messages (in my case 7000 messages), Kreya becomes unresponsive while attempting to display them in the "Responses" tab.

Furthermore, after restarting Kreya and navigating back to the method in question, the response is loaded again and Kreya again becomes unresponsive. Clicking "reset operation" on the method does not fix this.

**To Reproduce**
1. Connect to a gRPC service that has a streaming method that will return thousands of responses for a single request.
2. Send a request to the method.
3. Observe that Kreya is now unresponsive.

**Expected behavior**
Kreya should make some of the stream available to see in the UI, while hiding most of the data until it's requested/scrolled to/etc. Kreya should never become unresponsive during normal usage.

**Screenshots**
<img width="1512" alt="kreya-unresponsive" src="https://user-images.githubusercontent.com/19669836/222207489-daf9e7d0-aae8-42d8-bd6e-e45e2545a9bc.png">

**Environment (if possible, copy the information from the error dialog or the About menu):**
```json
{
  "kreyaVersion": "1.10.0",
  "releaseChannel": "stable",
  "osDescription": "Darwin 22.1.0 Darwin Kernel Version 22.1.0: Sun Oct  9 20:14:54 PDT 2022; root:xnu-8792.41.9~2/RELEASE_X86_64",
  "osVersion": "Unix 13.0.1",
  "osArch": "X64",
  "processArch": "X64",
  "runtimeIdentifier": "osx.13-x64",
  "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko)"
}
```

**Comment by CommonGuy** [comment]
Thanks for reporting this. Kreya currently cannot handle such a large amount of responses, as it tries to display them all at once. I added this to our backlog.

To "fix" this operation, you can purge the operation history cache Kreya -> Preferences, which makes the operation usable again.

**Comment by CommonGuy** [comment]
This has been implemented with Kreya v1.12
